onnxruntime/core/session/abi_session_options.cc

```diff
@@ -20,6 +20,42 @@ OrtSessionOptions::OrtSessionOptions(const OrtSessionOptions& other)
     : value(other.value), provider_factories(other.provider_factories) {
 }
 
+#if !defined(ORT_MINIMAL_BUILD) || defined(ORT_MINIMAL_BUILD_CUSTOM_OPS)
+onnxruntime::Status OrtSessionOptions::RegisterCustomOpsLibrary(onnxruntime::PathString library_name) {
```

Comment:
>onnxruntime::PathString [](http://example.com/codeflow?start=64&length=23)

Making a copy here seems redundant.

Reply:
The PathString is std::moved all the way down to the LibraryHandles where it is stored. So, this does not make any unnecessary copies.

If we're going to make a copy of this path string anyway, I find it better to have the function API inform the caller that the passed in string will be copied. If, on the other hand, this function accepted a `const ORTCHAR_T*` then the caller would not necessarily know if this string was going to be copied or if the pointer was just going to be stored in some member. imo